apps/slackbot/bolt_app.py

```diff
@@ -203,6 +229,27 @@ def slack_events():
     return handler.handle(request)
 
 
+@flask_app.route('/whitelist/add', methods=['POST'])
+def add_to_whitelist():
```

Comment:
Can we create a [router](https://flask.palletsprojects.com/en/2.3.x/quickstart/) for these? Just to make things more modularized